### PR TITLE
remove duplicated line for project key

### PR DESCRIPTION
**Pull request by philippeckel**
This commit just removes a duplicated linefor "project_key" in the documentation.

### CODE REVIEW

**Comment by peckeltw** [comment]
One more commit that renames 'token_url' to 'auth_url' in the documentation, which is more consistent with naming in Commercetools when creating an API client.

**Comment by peckeltw** [comment]
And one more that fixes the mount command for Docker

File: docs/index.md [hunk 1]
```diff
@@ -27,10 +27,9 @@ provider "commercetools" {
   client_id     = "<your client id>"
   client_secret = "<your client secret>"
   project_key   = "<your project key>"
-  project_key   = "<your project key>"
   scopes        = "<space seperated list of scopes>"
   api_url       = "<api url>"
-  token_url     = "<token url>"
+  auth_url      = "<auth url>"
```
Comment (mvantellingen) [inline docs/index.md:32]: But it currently is `token_url`. See https://github.com/labd/terraform-provider-commercetools/blob/master/commercetools/provider.go#L53

I kinda agree with naming it auth_url, but then we need to update the code first :-)

**Comment by mvantellingen** [comment]
Fixed the readme with the info provided. Thanks!

**Pull request closed without merging**